inference-engine/tests/functional/plugin/cpu/shared_tests_instances/subgraph_tests/variadic_split_pad.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "subgraph_tests/variadic_split_pad.hpp"
+
+using namespace SubgraphTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecision = {
+        InferenceEngine::Precision::FP32
+};
+
+const std::vector<InferenceEngine::SizeVector> shapes = {
+        {1, 8, 3, 2},
+        {3, 8, 8, 8},
+};
+
+const std::vector<std::vector<size_t>> connectedIndexes = {
+        {0},
+        {0, 2},
+        {0, 1, 3},
```

Comment:
Please add {0, 1, 1, 0} and {0, 0, 0, 1} schemes.

Reply:
`connectedIndexes` - is output indexes after Split to which Pad layer is attached. So if `{0}` - Pad is attached only to first Split output or if `{0, 2}` - to first and third outputs. `{0, 1, 1, 0}` and `{0, 0, 0, 1}` - are incorrect indexes

Reply:
It was discussed offline. I added these schemes. Thank you!